tests/e2e/test_simple_registry_workload.py

```diff
@@ -0,0 +1,45 @@
+import shlex
+from subprocess import Popen, PIPE
+from ocs_ci.framework.testlib import workloads
+import logging
+from ocs_ci.utility.svt import (
+    svt_project_clone, svt_create_venv_setup,
+    svt_cluster_loader, svt_cleanup
+)
+log = logging.getLogger(__name__)
+
+
+@workloads
+class TestRegistryWorkload:
+    def test_registry_workload(self, iterations=5):
+        """
+        https://github.com/openshift/svt/blob/master/openshift_performance/ose3_perf/scripts/build_test-README.md
+        The build_test.py scripts is a flexible tool for driving builds in OpenShift.
+        It can execute builds concurrently, sequentially, randomly or in arbitrary combinations.
+
+        Arguments:
+            -iterations (int): Number of iterations
+            -z : No-login(oc login)
+            -a : Run all builds in all projects
+            -n : Number of iterations (Do not use more than 10)
```

Comment:
The rest are not arguments of function , so please delete and put docstring before the place where it's used as a comment? Or if you would like to make it part of this docstring describe it as like 

```
Arguments for build_test.py:
    Describe your args here
```

Reply:
Done